src/NDItk/multigroup/ReactionCrossSections/test/ReactionCrossSections.test.cpp

```diff
@@ -0,0 +1,208 @@
+         "    16 1.1234567\n"
+         "    1 2 3 4 5\n"
+         "    6 7\n";
+}
+
+void verifyChunk( const ReactionCrossSections& chunk ) {
+
+  CHECK( false == chunk.empty() );
+  CHECK( 2 == chunk.numberReactions() );
+  CHECK( 7 == chunk.numberGroups() );
+
+  CHECK( true == chunk.hasReaction( 2 ) );
+  CHECK( true == chunk.hasReaction( 16 ) );
+  CHECK( false == chunk.hasReaction( 102 ) );
+
+  CHECK( 2 == chunk.reactions()[0].identifier() );
+  CHECK_THAT( 0.0, WithinRel( chunk.reactions()[0].qvalue() ) );
+  CHECK_THAT( 10.0, WithinRel( chunk.reactions()[0].crossSections()[0] ) );
+  CHECK_THAT( 20.0, WithinRel( chunk.reactions()[0].crossSections()[1] ) );
+  CHECK_THAT( 30.0, WithinRel( chunk.reactions()[0].crossSections()[2] ) );
+  CHECK_THAT( 40.0, WithinRel( chunk.reactions()[0].crossSections()[3] ) );
+  CHECK_THAT( 50.0, WithinRel( chunk.reactions()[0].crossSections()[4] ) );
+  CHECK_THAT( 60.0, WithinRel( chunk.reactions()[0].crossSections()[5] ) );
+  CHECK_THAT( 70.0, WithinRel( chunk.reactions()[0].crossSections()[6] ) );
+
+  CHECK( 16 == chunk.reactions()[1].identifier() );
+  CHECK_THAT( 1.1234567, WithinRel( chunk.reactions()[1].qvalue() ) );
+  CHECK_THAT( 1.0, WithinRel( chunk.reactions()[1].crossSections()[0] ) );
+  CHECK_THAT( 2.0, WithinRel( chunk.reactions()[1].crossSections()[1] ) );
+  CHECK_THAT( 3.0, WithinRel( chunk.reactions()[1].crossSections()[2] ) );
+  CHECK_THAT( 4.0, WithinRel( chunk.reactions()[1].crossSections()[3] ) );
+  CHECK_THAT( 5.0, WithinRel( chunk.reactions()[1].crossSections()[4] ) );
+  CHECK_THAT( 6.0, WithinRel( chunk.reactions()[1].crossSections()[5] ) );
+  CHECK_THAT( 7.0, WithinRel( chunk.reactions()[1].crossSections()[6] ) );
+
+  auto xs = chunk.reaction( 2 );
+  CHECK( 2 == xs.identifier() );
+  CHECK_THAT( 0.0, WithinRel( xs.qvalue() ) );
+  CHECK_THAT( 10.0, WithinRel( xs.crossSections()[0] ) );
+  CHECK_THAT( 20.0, WithinRel( xs.crossSections()[1] ) );
+  CHECK_THAT( 30.0, WithinRel( xs.crossSections()[2] ) );
+  CHECK_THAT( 40.0, WithinRel( xs.crossSections()[3] ) );
+  CHECK_THAT( 50.0, WithinRel( xs.crossSections()[4] ) );
+  CHECK_THAT( 60.0, WithinRel( xs.crossSections()[5] ) );
+  CHECK_THAT( 70.0, WithinRel( xs.crossSections()[6] ) );
+
+  xs = chunk.reaction( 16 );
+  CHECK( 16 == xs.identifier() );
+  CHECK_THAT( 1.1234567, WithinRel( xs.qvalue() ) );
+  CHECK_THAT( 1.0, WithinRel( xs.crossSections()[0] ) );
+  CHECK_THAT( 2.0, WithinRel( xs.crossSections()[1] ) );
+  CHECK_THAT( 3.0, WithinRel( xs.crossSections()[2] ) );
+  CHECK_THAT( 4.0, WithinRel( xs.crossSections()[3] ) );
+  CHECK_THAT( 5.0, WithinRel( xs.crossSections()[4] ) );
+  CHECK_THAT( 6.0, WithinRel( xs.crossSections()[5] ) );
+  CHECK_THAT( 7.0, WithinRel( xs.crossSections()[6] ) );
+}
+
+std::string chunkWithInsufficientNumberReactions() {
+
+  return "sig_reac\n";
+}
+
+std::string chunkWithInsufficientNumberCrossSectionValues() {
+
+  return "sig_reac\n"
+         "    2 0\n"
```

Comment:
Can you add some cross sections to one of these reactions? So it's like 2 groups for one, and 0 groups for the other.